### PR TITLE
Added LICENSE and improved the readme to reference oss stream-framework

### LINKED ISSUE #8: stream_django vs stream_framework
I am having a hard time determining the difference between these two. Is it simply that stream_django is a hosted (paid) version while stream_framework is not?

### PR DESCRIPTION
**Pull request by dwightgunning**
Fixes #8.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
+This example Django app shows you how you can easily build a site similar to Pinterest.
 
-### Demo
+The project is based on the [stream-django](https://github.com/GetStream/stream-django) integration for [Stream](https://getstream.io/). There is also a lower level [Python - Stream integration](https://github.com/getstream/stream-python) library which is suitable for all Python applications.
 
-You can [try the demo here](http://exampledjango.getstream.io).
-Alternatively you can deploy your own copy of the demo via Heroku.
+You can sign up for a Stream account at https://getstream.io/get_started.
+
+If you're looking to self-host your feed solution we suggest the open source <a href="https://github.com/tschellenbach/Stream-Framework" class="mktg-text-links">Stream-Framework</a>, created by the Stream founders.
```
Comment (hannesvdvreken) [inline README.md]: change this to `[]()` markdown syntax pls ;-)

Comment (hannesvdvreken) [reply README.md]: `cmd + K` <- protip

File: README.md [hunk 2]
```diff
@@ -76,3 +83,9 @@ The last bit of work is making sure the templates render nicely.
 This will render the templates in
 
 https://github.com/GetStream/Stream-Example-Py/tree/master/templates/activity
+
+## Copyright and License Information
+
+Copyright (c) 2014-2017 Stream.io Inc, and individual contributors. All rights reserved.
+
+See the file "LICENSE" for information on the history of this software, terms & conditions for usage, and a DISCLAIMER OF ALL WARRANTIES.
```
Comment (hannesvdvreken) [inline README.md]: maybe mention "bsd" somewhere

**Comment by hannesvdvreken** [comment]
Looking good! 👍 🎖

File: README.md (hunk 1 above)
Comment (dwightgunning) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (dwightgunning) [reply README.md]: Github detects the BSD-3 clause without the explicit title. I'll leave it like this for consistency (for now).

E.g: https://github.com/GetStream/stream-js/blob/master/LICENSE

**Comment by dwightgunning** [comment]
I've finalised adjustments based on @hannesvdvreken's feedback. Squashed the commit to keep the history tight.

@tbarbugli - any final comments? / Please merge.

**Pull request merged**